README.md

```diff
@@ -126,6 +130,7 @@ Every line starting by `m` starts a new mapping to the specified TO excel file.
 + `ToExcelIndex`: Which TO excel file that should be used. The first file has the index 0.
 + `FromSheetIndex`: The sheet number in the FROM excel file that should be used to lookup cells. The first sheet has the index 0.
 + `ToSheetIndex`: The sheet number in the TO excel file that should be used to lookup cells. The first sheet has the index 0.
++ `name`: user defined name for the mapping that will be displayed on error dialogues, may contain whitespace characters.
```

Comment:
to make it more exaplicit, I would use the term `mappingName` here (instead `name`)